src/ulong_extras/revbin.c

```diff
@@ -19,7 +19,7 @@
 # define HAVE_RBIT 1
 #endif
 
-#if defined(__GNUC__) && FLINT64 && HAVE_RBIT
+#if defined(__GNUC__) && FLINT64 && defined(__ARM_ACLE) && HAVE_RBIT
```

Comment:
```suggestion
#if FLINT64 && __ARM_ACLE && HAVE_RBIT
```

Actually, it says "When `__ARM_ACLE` is defined to 1, the header file is guaranteed to be available.", so it is possible that it is defined, but set to zero.  This also looks nicer ;-)